File: Backend/server/routes/photo.routes.js
const express = require('express');
const router = express.Router();
const controlador = require('../controller/photo.controller')

//imports
photoController = require('../controller/photo.controller')

// rutas
router.post('/subir/:id', controlador.uploadPicture);
router.post('/obtener/:id', controlador.getPhotos);

module.exports = router;

File: Backend/serverPython/routes/photo_routes.py
from flask import Blueprint, request

from serverPython.controller.photo_controller import uploadPicture
from serverPython.controller.photo_controller import getPhotos

photo_bp = Blueprint('foto', __name__, url_prefix='/foto')

photo_bp.add_url_rule('/subir/<id>', 'subirFoto', uploadPicture, methods=['POST'])
photo_bp.add_url_rule('/obtener/<id>', 'obtenerFoto', getPhotos, methods=['POST'])
